backend/logging/logrus.go

```diff
@@ -0,0 +1,7 @@
+package logging
+
+import "github.com/sirupsen/logrus"
+
+var logger = logrus.WithFields(logrus.Fields{
+	"component": "backend",
```

Comment:
Is that the right component name?

Reply:
I think so... I didn't want to name the component "logging" :D